cli/pkg/backend/aws/k8s_compute_backend.go

```diff
@@ -203,38 +206,128 @@ func (k8s *K8SComputeBackend) PrintLogs(ctx context.Context, stackName string, s
 	logrus.Infof("Found %d matching pods.", len(pods.Items))
 
 	for _, pod := range pods.Items {
-		logrus.Infof("... streaming logs from pod %s ...", pod.Name)
-
-		logs, err := k8s.ClientSet.CoreV1().Pods(k8s.HappyConfig.K8SConfig().Namespace).GetLogs(pod.Name, &corev1.PodLogOptions{
-			Follow: false,
-		}).Stream(ctx)
+		err = k8s.streamPodLogs(ctx, pod, false)
 		if err != nil {
-			return errors.Wrapf(err, "unable to retrieve logs from pod %s", pod.Name)
+			logrus.Error(err.Error())
 		}
-		defer logs.Close()
-		reader := bufio.NewScanner(logs)
+	}
+	return nil
+}
 
-		for reader.Scan() {
-			logrus.Info(string(reader.Bytes()))
-		}
-		logrus.Infof("... done streaming ...")
+func (k8s *K8SComputeBackend) streamPodLogs(ctx context.Context, pod corev1.Pod, follow bool) error {
+	logrus.Infof("... streaming logs from pod %s ...", pod.Name)
```

Comment:
I feel like these should be more debug/verbose messages

Reply:
I'd rather keep the source of logs; if not, one will end up hunting the logs by hand, this gives it a source of reference.